data/core/editor/help.cfg

```diff
@@ -122,151 +122,208 @@ Named locations can be accessed from WML using the Standard Location Filter’s
+This tool is only available in Scenario Mode, it adds WML <italic>text='[music]'</italic> tags to the scenario."
 [/topic]
+# wmllint: unbalanced-off
 # wmllint: markcheck on
 
 # wmllint: markcheck off
 [topic]
     id=..editor
     title= _ "Map/Scenario Editor"
-    # generator="contents:editor"
     text= "<img>src=icons/icon-editor.png align=left box=no float=yes</img>" + _ "Wesnoth’s Map and Scenario Editor allows users to create and edit the maps on which every Wesnoth scenario takes place. It also provides a limited set of features for setting up a basic scenario.
 
 The editor can be launched from the <italic>text='Map Editor'</italic> option at the title screen.
 
-<header>text='What you get'</header>
+<header>text='Editing Modes'</header>
 
-• <ref>dst='..editor_mode_terrain' text='Terrain Editor'</ref>
-An easy to use map editor, similar to simple paint applications.
+The editor features two modes of operation: terrain mode and scenario mode.
 
-• Scenario Editor
+The <ref>dst='..editor_mode_terrain' text='Terrain Mode'</ref> is similar to a simple paint application, with tools to <ref>dst='editor_tool_paint' text='paint'</ref>, <ref>dst='editor_tool_fill' text='fill'</ref>, <ref>dst='editor_tool_select' text='select (and copy)'</ref>, and <ref>dst='editor_tool_paste' text='paste'</ref>. It also has the tool for setting the leaders’ <ref>dst='editor_tool_starting' text='starting locations'</ref>.
 
-• <ref>dst='editor_playlist' text='Playlist Manager'</ref>
-Predefine the scenario’s music track playlist.
+The <ref>dst='..editor_mode_scenario' text='Scenario Mode'</ref>, in addition to the tools available in terrain mode, adds support for adding <ref>dst='editor_tool_label' text='labels'</ref>, <ref>dst='editor_tool_scenery' text='scenery items'</ref>, <ref>dst='editor_tool_unit' text='units'</ref> in addition to the leader, and assigning <ref>dst='editor_tool_village' text='village ownership'</ref> at the start of the scenario. There’s also a <ref>dst='editor_playlist' text='playlist manager'</ref> for the music.
 
-• Time Schedule Editor
+<bold>text='Warning: the Scenario Mode is buggy in 1.14.'</bold> Improving it is one of the main blockers for the 1.16 release, however in 1.14 its quality is far below that of the game and the terrain mode." + "
 
-<header>text='What you do *not* get'</header>
+" + _ "<header>text='What you do *not* get'</header>
 
 • What-you-see-is-what-you-get
-The editor is not a WYSIWYG application.
 
-Because which exact graphic tile represents a terrain in the map depends on all terrain rules loaded (which is different between the editor and each other use case) the map won’t look exactly the same.
+The editor is not a WYSIWYG application. Because which exact graphic tile represents a terrain in the map depends on all terrain rules loaded (which is different between the editor and each other use case) the map won’t look exactly the same in the game as it does in the editor.
```

Comment:
I don't think using the WYSIWYG acronym really adds anything useful here, or the "What-you-see-is-what-you-get" text above it.